File: RSPAClient/Core/AtPar.ViewModel/VM_MT_POU_CASE_CART_HEADER.cs


using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace AtPar.ViewModel
{
    public class VM_MT_POU_CASE_CART_HEADER
    {
        public List<VM_MT_POU_CASE_CART_HEADER_TB> lstCaseforQA { get; set; }
        public List<VM_MT_POU_CASE_CART_HEADER_TB> lstCaseInfo { get; set; }
        public List<VM_MT_POU_CASE_DISCRIPTON> lstDstCaseCartHeader { get; set; }
        public List<VM_MT_POU_CASE_DISCRIPTON> lstDescDistCase { get; set; }
        public List<VM_MT_POU_CASE_CART> lstCaseIdDesc { get; set; }
    }
}

File: RSPAClient/UserInterface/AtPar.Web/src/app/Entities/VM_MT_POU_CASE_CART_HEADER.ts
import { VM_MT_POU_CASE_CART_HEADER_TB } from "../../app/Entities/VM_MT_POU_CASE_CART_HEADER_TB";
import { VM_MT_POU_CASE_DISCRIPTON } from "../../app/Entities/VM_MT_POU_CASE_DISCRIPTON";
import { VM_MT_POU_CASE_CART } from "../../app/Entities/VM_MT_POU_CASE_CART";

export class VM_MT_POU_CASE_CART_HEADER {
    public lstCaseforQA: Array<VM_MT_POU_CASE_CART_HEADER_TB>;
    public lstCaseInfo: Array<VM_MT_POU_CASE_CART_HEADER_TB>;
    public lstDstCaseCartHeader: Array<VM_MT_POU_CASE_DISCRIPTON>;
    public lstDescDistCase: Array<VM_MT_POU_CASE_DISCRIPTON>;
    public lstCaseIdDesc: Array<VM_MT_POU_CASE_CART>;
}